ui/src/literature/components/ReferenceItem.jsx

```diff
@@ -73,10 +77,28 @@ class ReferenceItem extends Component {
           align="middle"
           className="w-100 sm-plus-flex-nowrap"
         >
-          <Col className="xs-sm-col-24">
-            {ReferenceItem.renderLabel(reference)}
+          <Col className="xs-sm-col-24 pr3">
```

Comment:
@karolina-siemieniuk-morawska wouldn't the `pr3` affect the UI if both flags are disabled?